mp4parse/src/boxes.rs

```diff
@@ -120,6 +120,7 @@ box_database!(
     ESDBox                            0x65736473, // "esds"
     VP8SampleEntry                    0x76703038, // "vp08"
     VP9SampleEntry                    0x76703039, // "vp09"
+    VP10SampleEntry                   0x76703130, // "vp10"
```

Comment:
It's tempting to ignore the VP10 variant, since the spec doesn't define anything about it beyond the name, and we're not planning to support it in Firefox. I suppose it does no harm though.